cli/installer/choco/azd.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>azd</id>
```

Comment:
The existing Chocolatey package id is 'azure-developer-cli' - https://community.chocolatey.org/packages/azure-developer-cli.

If you're really set on using 'azd' then it would be good to coordinate with me (as the maintainer of the existing package)